src/utils.cpp

```diff
@@ -70,19 +79,23 @@ void Logger::dumpState() {
        << _state.pmap_size << ","
        << _state.pmap_null_num << ","
        << _state.pmap_discard_num << ","
-       << getLogRemainingSpaceSize() << ","
+       << space_size << ","
        << dumpPrefixLens().c_str() << endl;
 }
 
+#ifdef GMP
 /*
  * Uses GMP library to dump a string version of the remaining state space size.
  * This number is typically very large (e.g. 10^20) which is why we use GMP instead of a long.
  * Note: this function may not work on some Linux machines.
  */
 std::string Logger::dumpRemainingSpaceSize() {
-    mpz_class s(_state.remaining_space_size);
-    return s.get_str();
+    char* str = mpz_get_str(NULL, 10, _state.remaining_space_size);
```

Comment:
Can we wrap in a ifdef GMP?

Reply:
Shouldn't this do the same thing? It's just getting the remaining state size as a string in base 10 format. This was we don't have to link with a whole new library (gmpxx)

Reply:
Ah sorry I missed the ifdef on line 86 -- looks good here!